### PR TITLE
Fixed Minor Copy-Paste Error in Manipulator Methods

### PR DESCRIPTION
**Pull request by 1000DollarBill**
In the openOuterManipulator and closeOuterManipulator functions there was a minor copy-paste error where it opened and closed the inner solenoid instead of the outer solenoid.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Couple things... 
1) add all your commands to the smartdash up in the constructor
2) add another command for dropping a cube (different than eject)
3) work with Dan and Tyler on exactly what piston open / close combinations should be done for which command (I don't think I was telling you the right thing)

**Comment by 1000DollarBill** [comment]
Fixed all your concerns Bill, I believe the pistons are OK now...

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java [hunk 1]
```diff
@@ -119,6 +124,14 @@ public Command OpenInnerManipulator() {
         return new ActionCommand("Open Inner Manipulator", this, this::openInnerManipulator);
     }
 
+    public Command CloseInnerManipulator() {
+        return new ActionCommand("Close Inner Manipulator", this, this::closeInnerManipulator);
+    }
+
+    public Command DropCube() {
```
Comment (ghost) [inline src/main/java/frc/team166/robot/subsystems/Manipulator.java]: Let's rename this to CubeDrop so it matches the other cube-related commands

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java [hunk 2]
```diff
@@ -119,6 +124,14 @@ public Command OpenInnerManipulator() {
...
+        return new ActionCommand("Drop Cube", this, this::openInnerManipulator);
+    }
+
     public Command CubePickup() {
```
Comment (ghost) [inline src/main/java/frc/team166/robot/subsystems/Manipulator.java]: Ben asked if this kind of thing will work: setting the motors to intake during initialization, and then not setting them in execute. Will that work? Or do you need to keep setting them in execute, not just in initialization?

**Review by ghost — CHANGES_REQUESTED** [review]
Two more minor things :)

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java (hunk 2 above)
Comment (bot190) [reply src/main/java/frc/team166/robot/subsystems/Manipulator.java]: When testing this today it did work correctly. I just looked and it turns out the motor safety stuff is only used on drive objects by default: [Motor Safety](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599705-using-the-motor-safety-feature)

**Comment by 1000DollarBill** [comment]
Fixed all comments and requested changes

**Pull request merged**